packages/happn-3/lib/client.js

```diff
@@ -940,7 +940,10 @@
   };
 
   HappnClient.prototype.__performSystemRequest = function (action, data, options, callback) {
-    var message = {
+    if (typeof callback !== 'function') {
```

Comment:
@southbite Please review this.  I think if we call perform system request with wrong arguments.  Callback isn't a function.  Think it's safe to fail early in this case.

Reply:
Is this necessary? Considering __performSystemRequest is internal we should perhaps expect it to always be called with the correct arguments?

Reply:
Yes, because when calling the client we have no idea how many arguments there might be.  e.g. changePassword with no arguments, means data will be in callback.  We could check every incoming method before it's wrapped, but that is alot more code.